server/jetstream_api.go

```diff
@@ -776,6 +776,28 @@ func (s *Server) sendAPIErrResponse(ci *ClientInfo, acc *Account, subject, reply
 	s.sendJetStreamAPIAuditAdvisory(ci, acc, subject, request, response)
 }
 
+const errRespDelay = 500 * time.Millisecond
+
+func (s *Server) sendDelayedAPIErrResponse(ci *ClientInfo, acc *Account, subject, reply, request, response string, rg *raftGroup) {
+	var quitCh <-chan struct{}
+	if rg != nil {
```

Comment:
I think I see often `rg.node != nil` check, so would add that too...

Reply:
added